index.html

```diff
@@ -266,5 +266,42 @@ <h3>Skills</h3>
         <div class="line"></div>
       </div>
     </section>
+
+    <section class="contact">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I&#39;d love to help
+        with it!
+      </p>
+
+      <form
+        action="https://formspree.io/f/xknaqwvy"
+        method="post"
+        class="contact-form"
+      >
+        <input
+          type="text"
+          class="name"
+          placeholder="name..."
+          class="placeholder"
+        />
+        <input
+          type="email"
+          class="email"
+          placeholder="email..."
+          class="placeholder"
+        />
+        <textarea
+          class="placeholder"
+          name="message"
+          id=""
+          cols="30"
+          rows="10"
+          placeholder="Write your message here..."
+        ></textarea>
```

Comment:
- [ ] According to the requirements user need to fill up every field for submission. Kindly use `required` attribute inside every field to protect blank submissions.
- Example: `<input type="text" id="username" name="username" required>`
 
![image](https://user-images.githubusercontent.com/60083437/228476026-00f58bf2-3121-436a-9355-b9f27654019b.png)